pom.xml

```diff
@@ -310,6 +300,48 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>3.2.1</version>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <!-- Will output 2 jars: the original, and the shaded one -->
+                            <shadedArtifactAttached>true</shadedArtifactAttached>
+
+                            <!-- final name of the shaded jar will be ${project.artifactId}-standalone -->
+                            <shadedClassifierName>standalone</shadedClassifierName>
+
+                            <relocations>
+                                <relocation>
+                                    <!-- Apache HttpClients -->
+                                    <pattern>org.apache.http</pattern>
+                                    <shadedPattern>unirest.shaded.org.apache.http</shadedPattern>
+                                </relocation>
+                                <relocation>
+                                    <!-- Jackson -->
+                                    <pattern>com.fasterxml.jackson</pattern>
+                                    <shadedPattern>unirest.shaded.com.fasterxml.jackson</shadedPattern>
+                                </relocation>
+                                <relocation>
+                                    <!-- Google Guava -->
+                                    <pattern>com.google.common</pattern>
+                                    <shadedPattern>shaded.com.google.common</shadedPattern>
```

Comment:
Eh this should be unirest.shaded.com.google.common for consistency but I missed it.